### DISCUSSION TITLE
GITHUB_TOKEN unauthenticated: User cannot be authenticated with the token provided.

### DISCUSSION THREAD

**Discussion by ghost** [community/community#137986] [Actions]
### Select Topic Area

Question

### Body

Hi all, currently I have an issue when I've tried to install dependencies from my own organization on a GH workflow

I've tried different ways, as you can see in the below picture

- Setup .npmrc (doing echo command with GH_TOKEN)
- Configure setup-node action with registry-url, scope and token

The steps are the following

![image](https://github.com/user-attachments/assets/887efb6d-c0c7-4b07-806f-25b080c676f7)

![image](https://github.com/user-attachments/assets/f30438b0-7559-430e-9b9a-d101e4834740)

If I edit the workflow adding the following permissions, the error changed as you can see in the below image

```
permissions:
  contents: read # for checkout
  packages: read # for npm ci
```

![image](https://github.com/user-attachments/assets/c93c3a15-a36c-44c2-a996-03c779ea3b84)

I attached an image of the packages from my organization

![image](https://github.com/user-attachments/assets/ddaf4d9f-08a2-4742-8f0d-765142968065)

I don't know what I'm doing wrong or if I missed some configuration. Someone could give me a hand?

I've read a possible solution is to create a new secret using a personal access token, but I'm not sure if it's a good one.

**Comment by zenkimoto** [comment]
@nicolasmastroviti - did you ever get an answer for this?  I'm struggling with the same.  Thanks!

**Reply by ghost** [reply]
Hi @zenkimoto

You need to go to your package (`Packages` tab within the organization) and enter to the `Package settings` as I show you on the capture below

![image](https://github.com/user-attachments/assets/547a9b26-fb2b-4dc3-93a6-dd4d5e01d24f)

After that, you need to add the repository 

![image](https://github.com/user-attachments/assets/f54e49d3-ceec-44f1-8d8a-1285a4953440)
